hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azure/ITestBlobDataValidation.java

```diff
@@ -65,21 +65,21 @@ public void tearDown() throws Exception {
    * Test that by default we don't store the blob-level MD5.
    */
   @Test
-  public void testBlobMd5StoreOffByDefault() throws Exception {
+  public void testBlobMd5StoreOffByDefault(TestInfo testInfo) throws Exception {
```

Comment:
we shoudn't need to do this. instead just capture once in setup and store

Reply:
Thank you for your suggestion! I will improve it.

Reply:
@steveloughran Could you please review this PR again? Thank you very much!